core/main/src/firebolt/handlers/advertising_rpc.rs

```diff
@@ -137,7 +134,23 @@ pub trait Advertising {
     #[method(name = "advertising.resetIdentifier")]
     async fn reset_identifier(&self, ctx: CallContext) -> RpcResult<()>;
 }
-
+const NONE: &str = "none";
+async fn get_advertisting_policy(platform_state: &PlatformState) -> AdvertisingPolicy {
+    AdvertisingPolicy {
+        skip_restriction: StorageManager::get_string(
+            platform_state,
+            StorageProperty::SkipRestriction,
+        )
+        .await
+        .unwrap_or_else(|_| String::from(NONE)),
+        limit_ad_tracking: StorageManager::get_bool(
```

Comment:
You may need to use the the following function to get limit_ad_tracking
`limit_ad_tracking: !PrivacyImpl::get_allow_app_content_ad_targeting(ps).await;`